Simplenote/Classes/SPOptionsViewController.m

```diff
@@ -313,6 +350,15 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
                     cell.accessoryView = self.biometrySwitch;
                     cell.tag = kTagTouchID;
                 }
+                case SPOptionsSecurityRowTimeout: {
+                    cell.textLabel.text = NSLocalizedString(@"Lock Timeout", @"Setting for when the passcode lock should enable");
+                    cell.selectionStyle = UITableViewCellSelectionStyleDefault;
+                    cell.accessoryView = nil;
+                    cell.tag = kTagTimeout;
+                    
+                    NSInteger timeoutPref = [[NSUserDefaults standardUserDefaults] integerForKey:kPinTimeoutPreferencesKey];
+                    [cell.detailTextLabel setText:timeoutPickerOptions[timeoutPref]];
+                }
```

Comment:
Same as above, missing `break;` on this one. (Nothing breaks though)

Reply:
Because of how we hide the `biometry` row, I actually needed to leave this break in here. I added a comment in the code. I'm open to alternative ideas :)